src/styles/base/base.scss

```diff
@@ -0,0 +1,39 @@
+@import '@/styles/variables.scss';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+  font-size: 16px;
+}
+
+.body {
+  color: $main-color;
+  font-size: 16px;
+  font-family: $font-family;
+  font-weight: 400;
+  letter-spacing: 1px;
+  background-color: $background-color;
+  min-height: 100vh;
+}
```

Comment:
I've already asked about vw. You use px, vh, vm, too many units. Please use only one type (I saw rem everywhere, it gives you full control over the size)